README.md

```diff
@@ -1 +1,62 @@
-# gluster-ansible-maintenance
\ No newline at end of file
+Role Name
+=========
+
+replace_node - Replace gluster node with a new node.
+
+Requirements
+------------
+
+- Ansible >= 2.6
+- lvm2 (Optional)
+
+
+NOTE: Inventory should contain a single node, which should be part of the cluster. This node should be different than any of the three variables explained below, this requirement is to collect the right peers for the new node.
+
+Role Variables
+--------------
+
+| Name                     |Choices| Default value         | Comments                          |
+|--------------------------|-------|-----------------------|-----------------------------------|
+| gluster_maintenance_old_node |    | UNDEF   | The node which has to be replaced with a new node. Just the node name is needed to get the peer id, the node need not be accessible. Provide the name that was used to probe the peers. |
+| gluster_maintenance_new_node |   | UNDEF | New node which will replace the old node. This name can be same as the old node or different name. |
+| gluster_maintenance_cluster_node |  | UNDEF | The node on which the peer, volume-id details are collected. This node should be part of the trusted storage pool. And should be different from old_node or new_node. |
+| gluster_maintenance_ovirt_url |  | UNDEF | URL for the ovirt management node |
```

Comment:
Is this mandatory? will the replace node work without the ovirt parameters?

Reply:
This is optional, since replace-node has to work both with hyperconvgerged and non-hyperconverged environments.

Reply:
Oh yes. I will do it right away, agree with that.